core/src/main/java/org/n52/iceland/request/AbstractServiceRequest.java

```diff
@@ -175,6 +180,18 @@ public Locale getRequestedLocale() {
         return LocaleHelper.fromString(getRequestedLanguage());
     }
 
+    public String getOriginalRequest() {
```

Comment:
Why do you not just return `Optional<String>`? `isSetOriginalRequest` wouldn't be needed that way....